android/src/main/cpp/NativeProxy.h

```diff
@@ -267,6 +269,8 @@ class NativeProxy : public jni::HybridClass<NativeProxy> {
 #ifdef RCT_NEW_ARCH_ENABLED
       jni::alias_ref<facebook::react::JFabricUIManager::javaobject>
           fabricUIManager,
+      std::function<void(std::function<void(jsi::Runtime &runtime)> &&callback)>
```

Comment:
use type RuntimeExecutor

Reply:
Done in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)